src/main/java/org/folio/innreach/batch/contribution/service/ContributionJobRunner.java

```diff
@@ -178,47 +180,51 @@ public void cancelContributionIfRetryExhausted(UUID centralServerId) {
     contributionService.cancelCurrent(centralServerId);
   }
 
-  public void runInstanceContribution(UUID centralServerId, Instance instance) {
-    log.info("Ongoing: validating instance {} for contribution to central server {}", instance.getId(), centralServerId);
-
-    boolean eligibleInstance = isEligibleForContribution(centralServerId, instance);
-    boolean contributedInstance = isContributed(centralServerId, instance);
-    log.info("Ongoing: eligibleInstance: {}, contributedInstance: {}", eligibleInstance, contributedInstance);
-    if (!eligibleInstance && !contributedInstance) {
-      log.info("Ongoing: skipping ineligible and non-contributed instance");
-      return;
-    }
-
-    runOngoing(centralServerId, (ctx, statistics) -> {
-      log.info("Ongoing: starting ongoing instance contribution job {}, centralServerId: {}", ctx, centralServerId);
-
+  public void runOngoingInstanceContribution(UUID centralServerId, Instance instance, OngoingContributionStatus ongoingContributionStatus) {
+    try {
+      log.info("runOngoingInstanceContribution:: validating instance {} for contribution to central server {}", instance.getId(), centralServerId);
+      boolean eligibleInstance = isEligibleForContribution(centralServerId, instance);
+      boolean contributedInstance = isContributed(centralServerId, instance);
+      log.info("runOngoingInstanceContribution:: eligibleInstance: {}, contributedInstance: {}", eligibleInstance, contributedInstance);
+      if (!eligibleInstance && !contributedInstance) {
```

Comment:
If these two can be null, perhaps we shouldn't log them in line 188. I think seeing them be null in the logs could be misconstrued as an error to be an error when it fact expected. Perhaps only log what is in 188 in an `else` of this `if`.

Reply:
Hi steve, After this if condition, we do have if, else if and else condition based on this Boolean flag. the logger added in those condition will take care of your concern I feel.